c++/measures/g4_iw.cpp

```diff
@@ -52,16 +53,18 @@ namespace cthyb {
 
     // Allocate temporary NFFT two-frequency matrix M
     {
-      int resize_factor = 3; // How much bigger should the large mesh bee???
-      int nfreq         = resize_factor * std::max(n_fermionic, n_bosonic - 1 + n_fermionic);
+      // Determine maximum Matsubara index needed
+      int nfreq = std::max(3 * n_fermionic, n_bosonic + n_fermionic);
       gf_mesh<imfreq> iw_mesh_large{beta, Fermion, nfreq};
+      // In case of fermionic measurement, large mesh only needed for first component
```

Comment:
I do not get the same result using a mesh with dimension {3*n_fermionic, n_fermionic}.

using {3*n_fermionic, 2*n_fermionic} works and {3*n_fermionic, 5*n_fermionic/3} as well but not smaller.

Where can I find the `fftw` oversampling factor inside of the `nfft` driver? I thing this should be increased a bit 
so that {3*n_fermionic, n_fermionic} gives the same result.

@Wentzell what do you think?